R/reduce_dimensions.R

```diff
@@ -2,12 +2,53 @@
 #'
 #' A new image with the modified width and height will be created
 #'
-#' @param input_file character
-#' @param output_file character
-#'
+#' @param input_file character, the file path for the input file
+#' @param output_file character, the file path for the output file
+#' @param width integer, the width of the reduced image
+#' @param height integer, the height of the reduce image
 #' @return an array of image
 #' @export
-#' @examples
+#' @examples reduce_dimensions("images/mandrill.jpg", "images/reduced.jpg", 297, 200)
 #' reduce_dimensions(input_file,output_file,width,height)
-reduce_dimensions <- function(input_file,output_file,width,height) {
+library("jpeg")
+library(testthat)
+reduce_dimensions <- function(input_file, ouput_file, width, height) {
+  img <- readJPEG(input_file)
```

Comment:
```suggestion
  img <- jpeg::readJPEG(input_file)
```
suggest load package by :: instead of attaching the library. As far as I can see, there's only two places that used jpeg library. And does the function need testthat?